### PR TITLE
Add testdir to github action cache and increase cryptocompare retries

### LINKED ISSUE #873: Cryptocompare query failed with "You are over your rate limit, please upgrade your account"
## Problem Definition

This happened in the tests here: https://github.com/rotki/rotki/pull/872/checks?check_run_id=581306979

```
>                   raise RemoteError(error_message)
E                   rotkehlchen.errors.RemoteError: Failed to query cryptocompare for: "https://min-api.cryptocompare.com/data/v2/histohour?fsym=DASH&tsym=EUR&limit=2000&toTs=1434470400". Error: You are over your rate limit please upgrade your account!
```

## Task

Find out what's this new error from cryptocompare and handle accordingly.

Edit: What happened is that all 4 retries during the backing off were used up and then Rotki gave up querying cryptocompare. Perhaps ... increase the number of retries?

### PR DESCRIPTION
**Pull request by LefterisJP**
Fix #873

### CODE REVIEW

File: .github/workflows/rotki_ci.yml [hunk 1]
```diff
@@ -176,6 +176,12 @@ jobs:
           key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
           restore-keys: |
             ${{ runner.os }}-node-
+      - uses: actions/cache@v1
+        with:
+          path: ~/.cache/.rotkehlchen-test-dir
```
Comment (kelsos) [inline .github/workflows/rotki_ci.yml:181]: Just a question is that directory also used in OSX or does OSX uses `~/Library/Caches` instead

Comment (LefterisJP) [reply .github/workflows/rotki_ci.yml:181]: It's used in OSX too

Comment (LefterisJP) [reply .github/workflows/rotki_ci.yml:181]: @kelsos if it's also used in OSX this will work, right?

Comment (kelsos) [reply .github/workflows/rotki_ci.yml:181]: Yes, I read the description in the issue after asking the question. You seem to put it in ~/.cache in OSX too so I don't see why it won't work

Comment (LefterisJP) [reply .github/workflows/rotki_ci.yml:181]: thanks!

**Pull request merged**